pushy-server-schema/Makefile

```diff
@@ -8,14 +8,17 @@ setup_schema:
 	@echo "Dropping and recreating database '$(TEST_DB)'"
 	@psql --dbname template1 --single-transaction --command 'DROP DATABASE IF EXISTS $(TEST_DB)'
 	@createdb $(TEST_DB)
-#	@psql --dbname $(TEST_DB) --single-transaction -f /Users/cm/oc/code/opscode/opscode-pushy-server/db/pgsql_schema.sql
 	@sqitch --engine pg --db-name $(TEST_DB) deploy
 
 setup_tests:
-	@psql --dbname $(TEST_DB) --command 'CREATE EXTENSION pgtap;'
-	@psql --dbname $(TEST_DB) --command 'CREATE EXTENSION chef_pgtap;'
+	#@psql --dbname $(TEST_DB) --command 'CREATE EXTENSION pgtap;'
```

Comment:
Need to figure out what happened here - this was working fine in local docker runs until suddenly it wasn't.